shared/chat/conversation/messages/wrapper/shared.js

```diff
@@ -12,20 +12,82 @@ const LeftMarker = ({author, isYou, isFollowing, isBroken}) => (
   <Box style={{..._leftMarkerStyle, backgroundColor: marginColor(author, isYou, isFollowing, isBroken)}} />
 )
 
-const UserAvatar = ({author, showImage}) => (
-  <Box style={_userAvatarStyle}>
-    {showImage && <Avatar size={24} username={author} skipBackground={true} />}
-  </Box>
-)
+type UserAvatarProps = {
+  author: string,
+  showImage: boolean,
+  onClick: (author: string) => void,
+}
+
+class UserAvatar extends PureComponent<void, UserAvatarProps, void> {
+  _onClick: () => void
+
+  constructor(props: UserAvatarProps) {
+    super(props)
+
+    this._updateOnClick(props)
+  }
 
-const Username = ({author, isYou, isFollowing, isBroken, includeHeader}) => {
-  if (!includeHeader) return null
-  const style = {
-    color: colorForAuthor(author, isYou, isFollowing, isBroken),
-    ...(isYou ? globalStyles.italic : null),
-    marginBottom: 2,
+  _updateOnClick(p: UserAvatarProps) {
+    const {onClick, author} = p
+    this._onClick = () => {
```

Comment:
Changed these two back to dumb components using ClickableBox, so this is moot I think.